privacy_estimates/binomial_proportion.py

```diff
@@ -6,15 +6,13 @@
 from privacy_estimates.privacy_region import eps_from_fnr_fpr, delta_from_fnr_fpr
 
 
-def compute_binomial_proportion(count: AttackResults, alpha: float, method: str) -> Tuple[float, float, float, float]:
+def compute_confidence_region(count: AttackResults, alpha: float, method: str) -> Tuple[float, float, float, float]:
```

Comment:
I think this name can be confusing; we have a privacy region and this is not the same kind of region (it has also a particular geometry). I'm out of ideas.

Reply:
Let's name it exactly what it does: `compute_fnr_fpr_confidence_intervals`